design/features/payment-links/README.md

```diff
@@ -0,0 +1,58 @@
+# Add a payment link
+
+## Status
+
+Date created: *2024-04-29* 
+
+Epic trello card: https://trello.com/c/7S8AarCY/1430-epic-add-a-payment-link  
+Mural working board: https://app.mural.co/t/gaap0347/m/gaap0347/1697117881057/d45c5ac653b9730a76915e77f257bacdcbb40047?sender=ue1ef9fc3c2ad3697c3c93132  
+___
+
+## Contents  
+
+- [Add a payment link](#add-a-payment-link)
+  - [Status](#status)
+  - [Contents](#contents)
+  - [What](#what)
+    - [User stories](#user-stories)
+  - [Why](#why)
+  - [Hypothesis](#hypothesis)
+___
+
+## What  
+
+This feature introduces new functionality to the form building tool to allow form creators to take a payment for their service. This way of taking payments is utilising the [payment page](https://payments.service.gov.uk/govuk-payment-pages/), or payment link, feature developed by GOV.UK Pay. This feature allows form fillers to pay for the service after submitting their form, making it quicker and easier to get the outcome they need.  
+
+
+### User stories  
+
+**As a form creator**, I need to request payment, so that I can provide the service to the form filler.  
+**As a form creator**, I need to know how to set up payments, so I can take the relevant steps.   
+
+**As a form filler**, I need to be able to pay, so that I can get the outcome in a timely manner.  
+**As a form filler**, I need to clearly know what to do next after completing the form, so I don’t waste effort.  
+**As a form filler**, I need to get a confirmation that my payment has been received.  
+**As a form filler**, I need a copy of the payment confirmation, so I have proof that I've completed what I needed to and don’t have any issues proving this later on.  
+**As a form filler**, I need to know what part of the process I’m doing, so that I know I’m doing the right thing.  
+
+**As a form processor**, I need to be able to reconcile who has filled a form and who has paid.  
+
+**As a department**, I need to collect the money owed for the services we provide, so we can cover costs of running an effective service (and clear our debt).  
+**As a department**, I need to know when to set up a Pay account and how to do it, so it is in place by the time we need it.  
+
+
+## Why  
+
+When users fill in a form they may often be asked for a payment for the service. This can be as a Bacs (Bankers Automated Clearing System) payment, cheque or may be followed up with a link or phone call to request payment. This can make the process take longer meaning the user doesn’t receive an answer or outcome in a timely manner. This process can also mean more work for internal staff as they may need to chase for payment or make follow up calls before they can proceed.  
```

Comment:
```suggestion
Many document-based forms also require a payment for the service. This can be as a Bacs (Bankers Automated Clearing System) payment, cheque or may be followed up with a link or phone call to request payment. This can make the process take longer meaning the user doesn’t receive an answer or outcome in a timely manner. This process can also mean more work for internal staff as they may need to chase for payment or make follow up calls before they can proceed.  
```
Just to make it clearer we're taking about a feature we needed to be able to digitise document-based forms?